### PR TITLE
CI: Fix workflow-example

### PR DESCRIPTION
**Pull request by eloycoto**
Fix workflow-example CI test and also fail if any command fails

### CODE REVIEW

File: workflow-examples/run_examples.sh [hunk 1]
```diff
@@ -89,6 +92,24 @@ get_next_workflow() {
     -H 'Content-Type: application/json' | jq '.tasks[] | select(has("nextWorkFlow"))' | jq -r '.nextWorkFlow' | head -n 1
 }
 
+wait_project_start() {
+  echo_blue "******** Checking project is running ********"
+  for i in {1..1}
```
Comment (masayag) [inline workflow-examples/run_examples.sh:97]: `1..1` is a single iteration, right?

Comment (masayag) [reply workflow-examples/run_examples.sh:97]: meaning, `i` isn't progressing, so only one iteration and the condition on 108 will always be false?

Comment (eloycoto) [reply workflow-examples/run_examples.sh]: sorry, was testing it and I forget it.

**Review by RichardW98 — APPROVED** [review]

**Review by pkliczewski — COMMENTED** [review]
/lgtm

**Pull request merged**